### PR TITLE
【リファクタ】Mockフレーバーの設定

### PR DESCRIPTION
**Pull request by wasabi-muffin**
# 概要
・data/repositoryのmockファイルを追加した
・appのmockのdiを設定した
・buildSrc・gradleの不要なもの削除した

# 変更種類
- [ ] フィーチャー実装
- [ ] バッグ修正
- [ ] リファクタ
- [ ] ドキュメンテーション
- [ ] その他（）

# テスト
- [ ] Unit Test
- [ ] UI Test
- [ ] 動作確認
- [ ] N/A

### CODE REVIEW

File: app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package jp.co.yumemi.android.code_check.di
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ActivityRetainedComponent
+import jp.co.yumemi.android.code_check.data.repositories.RepositoriesDataRepository
```
Comment (gmvalentino8) [inline app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt:7]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt#L7"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import jp.co.yumemi.android.code_check.data.repositories.RepositoriesMockRepository
+import jp.co.yumemi.android.code_check.data.repositories.SearchMockRepository
+import jp.co.yumemi.android.code_check.data.sources.RepositoriesRemoteDataSource
```
Comment (gmvalentino8) [inline app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt:10]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt#L10"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import jp.co.yumemi.android.code_check.data.sources.SearchLocalDataSource
```
Comment (gmvalentino8) [inline app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt:11]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt#L11"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import jp.co.yumemi.android.code_check.data.sources.SearchRemoteDataSource
```
Comment (gmvalentino8) [inline app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt:12]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/app/src/mock/kotlin/jp/co/yumemi/android/code_check/di/DataModule.kt#L12"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package jp.co.yumemi.android.code_check.data.repositories
+
+import jp.co.yumemi.android.code_check.data.mappers.PullRequestDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt:3]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt#L3"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.ReleaseDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt:4]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt#L4"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.UserDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt:5]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt#L5"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+import jp.co.yumemi.android.code_check.data.sources.RepositoriesRemoteDataSource
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt:6]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt#L6"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+import jp.co.yumemi.android.code_check.domain.entities.PullRequest
+import jp.co.yumemi.android.code_check.domain.entities.Release
+import jp.co.yumemi.android.code_check.domain.entities.User
+import jp.co.yumemi.android.code_check.domain.repositories.RepositoriesRepository
+
+class RepositoriesMockRepository: RepositoriesRepository {
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt:12]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/RepositoriesMockRepository.kt#L12"></span>Missing spacing before ":"</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 10]
```diff
@@ -0,0 +1,49 @@
+package jp.co.yumemi.android.code_check.data.repositories
+
+import jp.co.yumemi.android.code_check.data.mappers.IssueDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:3]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L3"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.OrganizationDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:4]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L4"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.PullRequestDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:5]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L5"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.RecentSearchDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:6]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L6"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.RepositoryDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:7]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L7"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.mappers.UserDataMapper
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:8]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L8"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.sources.SearchLocalDataSource
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:9]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L9"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+import jp.co.yumemi.android.code_check.data.sources.SearchRemoteDataSource
```
Comment (gmvalentino8) [inline data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt:10]: <table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/a4ac01f77f1e88ee58a43265359d64c729c51332/data/src/mock/kotlin/jp/co/yumemi/android/code_check/data/repositories/SearchMockRepository.kt#L10"></span>Unused import</td>
    </tr>
  </tbody>
</table>

**Comment by wasabi-muffin** [comment]
No error was reported but at least one warning was found.

<p align="right" data-meta="generated_by_./Dangerfile">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**